code/modules/jobs/job_types/_job.dm

```diff
@@ -127,6 +127,10 @@
 	/// You'll probably break someone's config if you change this, so it's best to not to.
 	var/config_tag = ""
 
+
+	/// department's winter coat
+	var/dept_jacket = /obj/item/clothing/suit/hooded/wintercoat
```

Comment:
not really a departmental jacket if its a job specific winter coat, some other variable name would be cool
or like you can just make a static list of job to coat, like the cybernetic revolution station trait. you just need to add it to the unit test for station traits

Reply:
you should do that route instead

Reply:
how where would i do that

Reply:
just look at /datum/station_trait/cybernetic_revolution

Reply:
i feel like getting the job's jacket could be useful as like a costume thing tho ig, coud i just rename the var